src/assets/css/app.scss

```diff
@@ -1 +1,2 @@
 @import "panels";
+@import "idea";
```

Comment:
Is it possible to import from path? Tried `../../../node_modules/highlight.js/styles/idea.css`.